src/app/core/common-components/description-only/edit-description-only/edit-description-only.component.spec.ts

```diff
@@ -8,6 +9,7 @@ describe("EditDescriptionOnlyComponent", () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       imports: [EditDescriptionOnlyComponent],
+      providers: [ComponentRegistry],
```

Comment:
is this still needed to be added?

Reply:
Yes, it is needed in the test because ComponentRegistry is required by the component tree related to MarkdownPageModule, which we are using in EditDescriptionOnlyComponent.
without `providers: [ComponentRegistry]`, it is showing `NullInjectorError: NullInjectorError: No provider for ComponentRegistry!`